js/tji-datasets-explore.js

```diff
@@ -119,31 +118,38 @@ TJIGroupByBarChart.prototype.get_options_overrides = function() {
   return {};
 }
 
-// Group the data by the groupBy key, and count how many records
-// have each key. Returns an object with two lists:
+// Group the data by the groupBy key, 
+// and create collection of key and record count for each group
+// sort this collection by this.sort_by -- this impacts the colors for each group
```

Comment:
I don't think we need to explain as much of the internal workings in this comment, simply what the function does at a high level. Suggested change:
```
Group the records by the value in their this.group_by field,
and count how many records have each value.
Use this.sort_by to determine how to order them.
Returns an object with two lists...
```

Reply:
I've got something like that now